### DISCUSSION TITLE
在主鍵特性設置自增的情況下，在插入數據時手動設置主鍵的值，插入后的數據，主鍵的值不是設置的值，而是自增的值

### DISCUSSION THREAD

**Discussion by xbin-y** [dotnetcore/FreeSql#947] [General]
特性：
[JsonProperty, Column(Name = "id", IsPrimary = true, IsIdentity = true)]

mysql,sqlite 在手寫sql的情況下，是支持自增字段手動插入值的

**Comment by 2881099** [comment]
fsql.Insert(item).InsertIdentity().ExecuteAffrows();
